full-node/sov-stf-runner/src/runner.rs

```diff
@@ -137,7 +152,7 @@ where
     }
 
     /// Runs the rollup.
-    pub async fn run_in_process(&mut self) -> Result<(), anyhow::Error> {
+    pub async fn run_in_process(&mut self, zk_storage: V::PreState) -> Result<(), anyhow::Error> {
```

Comment:
Sorry, I was not clear can this be passed as a field in ` StateTransitionRunner` ?